config.py

```diff
@@ -141,6 +30,22 @@ def __init__(self):
         if not os.path.exists(self.captionsFolder):
             os.makedirs(self.captionsFolder)
 
+        self.videoToUse = ''
+
+        self.saveFolder = "savedData"
+        self.fileTypes = ["topicModel", "topicsOverTime"]
+
+        for folder in self.fileTypes:
+            folderPath = os.path.join(self.saveFolder, folder)
+            if not os.path.exists(folderPath):
+                os.makedirs(folderPath)
+
+        self.representationModelType = "langchain"
+        self.useKeyBERT = True
```

Comment:
Can you explain what is the purpose of this variable `useKeyBERT`?

Reply:
It's a variable that toggles the usage of the KeyBERT vectorizer in the BERTopic model. I kept is as an option if we ever want to toggle it. This will also be added to the .env file, defaulting to True.

Reply:
I have added this is a toggle option in the .env file to also explain what it does,